config/supervisor/webhook/webhookcainjection_patch.yaml

```diff
@@ -0,0 +1,15 @@
+# This patch add annotation to admission webhook config and
+# the variables $(NAMESPACE) and $(CERTIFICATENAME) will be substituted by kustomize.
```

Comment:
Do we have this patch now twice? Are we not also importing the one from config/base?

I wonder if we should additionaly move the one from config/base to config/webhook (seems cleaner)

Reply:
Seems like `config/base` is just patching `config/webhook` as only `config/webhook` is listed as resources in base/kustomization.yaml. And config/supervisor/webhook is patching separately.

I think it's good to move it from config/base to config/webhook, it's more clear. I'm afraid we still need to keep `config/supervisor/webhook`, otherwise we need to add it as a resource of config/webhook, and this will cause recursed reference and problem with current structure.